srcCpp/perftest_publisher.cxx

```diff
@@ -7,6 +7,7 @@
 #include "RTIDDSImpl.h"
 #include "perftest_cpp.h"
 #include "CpuMonitor.h"
+#include <vector>
```

Comment:
Include vector in "perftest_cpp.h" instead of here.

Reply:
This would be cleaner. But I think is a good practice to have all the includes that a .cpp is used in the same file.

Reply:
Sure, I can do it.

As @alfonso-soto said, I had as common practice to include in the .h whatever is used in the .h and then include the rest needed for the implementation in the .cpp file.

I will change it on my next commit.